tests/unit/test_work_item_relation_definitions.py

```
@@ -0,0 +1,62 @@
+        try:
+            updated = client.work_item_relation_definitions.update(
+                workspace_slug,
+                created.id,
+                UpdateWorkItemRelationDefinition(outward="depends on"),
+            )
+            assert updated.id == created.id
+            assert updated.outward == "depends on"
+        finally:
+            try:
+                client.work_item_relation_definitions.delete(workspace_slug, created.id)
+            except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)